### PR TITLE
Fix: Nested Tabs - Align Title isn't working well with Icon Position Top and Bottom [ed-8915]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/assets/scss/frontend.scss [hunk 1]
```diff
@@ -84,9 +86,9 @@
 
 		&-title {
 			display: flex;
-			align-items: var( --n-tabs-title-align-items );
+			align-items: var( --n-tabs-title-align-items-toggle, var( --n-tabs-title-align-items ) );
```
Comment (davseve) [inline modules/nested-tabs/assets/scss/frontend.scss:89]: Why do we need the fallback if `--n-tabs-title-align-items-toggle` always contains a value?
I see that `--n-tabs-title-align-items-toggle` is set in row 42.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js [hunk 2]
```diff
@@ -326,6 +326,63 @@ test.describe( 'Nested Tabs tests', () => {
...
+			tabsId = await editor.addWidget( 'nested-tabs', container );
+
+		await editor.getPreviewFrame().waitForSelector( '.e-n-tabs-content .e-con.e-active' );
+
+		// Act.
+		// Add Icons.
+		await setIconsToTabs( page, TabsIcons );
+		const activeTab = editor.getPreviewFrame().locator( '.e-normal.e-active' );
+
+		// Tabs styling scenario 1: Direction: Top, Align Title: Left, Icon Position: Right.
```
Comment (davseve) [inline tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:345]: I loved the new associative styling of the test,
More readable.

**Review by davseve — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/assets/scss/frontend.scss (hunk 1 above)
Comment (hein-obox) [reply modules/nested-tabs/assets/scss/frontend.scss:89]: @davseve Yes, we assign `initial` to `--n-tabs-title-align-items-toggle` it use the fallback value instead.

File: tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js (hunk 2 above)
Comment (hein-obox) [reply tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js:345]: Nice.

**Review by davseve — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Review by TzviRabinovitch — APPROVED** [review]

**Pull request merged**